src/lib/ngsiNotify/senderThread.h

```diff
@@ -61,6 +61,7 @@ typedef struct SenderThreadParams
   std::string                        fiwareCorrelator;
   struct timespec                    timeStamp;
   std::map<std::string, std::string> extraHeaders;
+  std::string                        subscriptionId;
```

Comment:
Yeah, afraid it is needed:
In `startSenderThread()`, when `httpRequestSend()` fails, `subCacheItemNotificationErrorStatus()` is called with params->subscriptionId (and tenant), so that subCacheItemNotificationErrorStatus can look up the subscription in the cache and update the item with error counter / tiemstamp.